mcp_server/pkg/internalapi/config.go

```diff
@@ -67,6 +71,7 @@ type Config struct {
 	Loghub2Endpoint      string
 	UserEndpoint         string
 	RBACEndpoint         string
+	FeatureHubEndpoint   string
```

Comment:
@DamjanBecirovic don't see this config value being updated in `LoadConfig()` below

Reply:
I've fixed this issue, added tests, and verified that it works on the preprod environment.